.github/workflows/ci.yaml

```diff
@@ -25,6 +25,24 @@ jobs:
         with:
           go-version: 1.18
       - run: go build -v ./...
+      - name: "verify secrets"
+        shell: bash
+        env:
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENAI_ORG_ID: ${{ secrets.OPENAI_ORG_ID }}
+        run: |
+          if [[ -z "${OPENAI_API_KEY}" ]]; then
```

Comment:
try moving that env block under the `with`, in line with `go-version` 

https://github.com/containers/fetchit/blob/main/.github/workflows/docker-image.yml#L1129-#L1130
```
with:
  env:
```
maybe? still looking thru docs, but you can try that

Reply:
or, try putting the env block above the steps block, in line with runs-on